### PR TITLE
graphviz@8.0.4: add separate shims for different layout engines

### LINKED ISSUE #4428: Question on the contents of the graphviz package
This package distributes multiple executables besides the usual `dot` - `neato`, `twopi`, `fdp` and so on.

Essentially, they provide another default layout for `dot` depending on the name of the executable. I found that back in 2020 (commit 453df2929ffb35caa64b7954fa7690a5a1f66a28) these executables were removed from the manifest. The commit message was not really clear on why exactly they were removed.

I would like to re-add these executables in the form of symlinks/shims as specified in the documentation.

The full list of such layouts:
- circo
- fdp
- neato
- osage
- patchwork
- sfdp
- twopi

I wonder if there are drawbacks or objections to that approach. I guess I'll manage a PR in the coming days.

### PR DESCRIPTION
**Pull request by ShrykeWindgrace**
Closes #4428

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by HUMORCE** [comment]
docs?

**Comment by ShrykeWindgrace** [comment]
@HUMORCE Could you elaborate? What docs do you want to update?

File: bucket/graphviz.json [hunk 1]
```diff
@@ -30,7 +30,14 @@
...
         "bin\\tred.exe",
-        "bin\\unflatten.exe"
+        "bin\\unflatten.exe",
+        ["bin\\dot.exe", "circo"],
+        ["bin\\dot.exe", "fdp"],
+        ["bin\\dot.exe", "neato"],
+        ["bin\\dot.exe", "osage"],
+        ["bin\\dot.exe", "patchwork"],
+        ["bin\\dot.exe", "sfdp"],
+        ["bin\\dot.exe", "twopi"]
```
Comment (HUMORCE) [inline bucket/graphviz.json:40]:
```suggestion
        "bin\\unflatten.exe",
        [
            "bin\\dot.exe",
            "circo"
        ],
        [
            "bin\\dot.exe",
            "fdp"
        ],
        [
            "bin\\dot.exe",
            "neato"
        ],
        [
            "bin\\dot.exe",
            "nop2",
            "-Knop2"
        ],
        [
            "bin\\dot.exe",
            "osage"
        ],
        [
            "bin\\dot.exe",
            "patchwork"
        ],
        [
            "bin\\dot.exe",
            "sfdp"
        ],
        [
            "bin\\dot.exe",
            "twopi"
        ]
```

**Comment by HUMORCE** [comment]
I mean the docs you mentioned in #4428, I already found it: https://graphviz.org/docs/layouts/

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**